app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -208,6 +219,21 @@ class AddDeadlineActivity : AppCompatActivity() {
             }
         }
     }
+    private val startForResult =
```

Comment:
As we search a pdf, could this be tested ?

Reply:
I tried. Haven't found yet how to do it